dhis-2/dhis-api/src/main/java/org/hisp/dhis/common/ValueType.java

```diff
@@ -78,7 +77,7 @@
     URL( String.class, false ),
     FILE_RESOURCE( String.class, true, FileTypeValueOptions.class ),
     IMAGE( String.class, false, FileTypeValueOptions.class ),
-    GEOJSON( GeoJSON.class, false ),
+    GEOJSON( String.class, false ),
```

Comment:
Just to highlight: This was needed because using GeoJSON here can become an issue with reflection from/in modules that do not have the dependency on the geotools artefact. Also this had no actual function apart from controlling the header in the grid view of values.